package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@carforyou/phraseapp",
-  "version": "0.0.2",
+  "version": "0.1.0-1",
```

Comment:
What does the `-1` mean here? Build number of pre-release version?

Reply:
yes! prerelease!